packages/cactus-test-tooling/src/main/typescript/openethereum/openethereum-test-ledger.ts

```diff
@@ -178,20 +191,67 @@ export class OpenEthereumTestLedger {
    * Creates a new ETH account from scratch on the ledger and then sends it a
    * little seed money to get things started.
    *
+   * Uses `web3.eth.accounts.create`
+   *
    * @param [seedMoney=10e8] The amount of money to seed the new test account with.
    */
   public async createEthTestAccount(seedMoney = 10e8): Promise<Account> {
-    const fnTag = `${this.className}#getEthTestAccount()`;
+    const ethTestAccount = this.web3.eth.accounts.create(uuidv4());
+
+    const receipt = await this.transferAssetFromCoinbase(
+      ethTestAccount.address,
+      seedMoney,
+    );
+
+    if (receipt instanceof Error) {
+      throw receipt;
```

Comment:
@outSH I think if you re-throw it like this then you chop off the previous call stack and then start a new one from this point. I'd recommend re-throwing a `RuntimeError` [1] that wraps the original error and also adds some contextual detail about what was being done and where when something went wrong.

[1]: https://www.npmjs.com/package/run-time-error

Reply:
@petermetz Done.